docs/README.md

```diff
@@ -1,6 +1,6 @@
 # Website
 
-This website is built using [Docusaurus 2](https://docusaurus.io/), a modern static website generator.
+This website is built using [Docusaurus](https://docusaurus.io/), a modern static website generator.
```

Comment:
lol, this makes me wonder if there was a major distinction between v1 and v2 for Docusaurus or something

Reply:
lol, not sure, this was part of Docusaurus updates when I generated a fresh docs to replace the old version